cspell.json

```diff
@@ -36,7 +36,9 @@
     "workletize",
     "workletized",
     "workletizes",
-    "worklets"
+    "worklets",
+    "androidx",
+    "appcompat"
```

Comment:
Local CI was failing on those two.

Reply:
@maciekstosio please try keeping the `cspell.json` file in an alphabetical order 😄

Reply:
Sure, fixed!